doc/mgr/dashboard.rst

```diff
@@ -257,3 +260,164 @@ to use hyperlinks that include your prefix, you can set the
   ceph config set mgr mgr/dashboard/url_prefix $PREFIX
 
 so you can access the dashboard at ``http://$IP:$PORT/$PREFIX/``.
+
+
+User and Role Management
+------------------------
+
+User Accounts
+^^^^^^^^^^^^^
+
+Ceph Dashboard supports the management of multiple user accounts. Each user
+account is composed by a username, a password (stored in ecrypted form using
+``bcrypt``), an optional name, and an optional email.
+
+User accounts are stored in MON's configuration database, and are globaly
+shared across all ceph-mgr instances.
+
+We provide a set of CLI commands to manage user accounts:
+
+- *Show User(s)*::
+
+  $ ceph dashboard ac-user-show [<username>]
+- *Create User*::
+
+  $ ceph dashboard ac-user-create <username> <password> [<name>] [<email>]
```

Comment:
Would it be possible to maintain the original `ceph dashboard set-login-credentials` command for backwards compatibility reasons? This would help in backporting this to Mimic at some point.

Reply:
Yes, I'll add that in a new commit

Reply:
re-added the `set-login-credentials` command